verrazzano/definer.go

```diff
@@ -0,0 +1,104 @@
+// We need the word "Copyright" here so this file will pass the license check
```

Comment:
put real notices in please

Reply:
did we write this? or copy it from somewhere?  if we copied it, we need to include attributions

Reply:
it looks copied to me

Reply:
I confirmed with Jeff Berkowitz that he wrote these modules.